### ISSUE TITLE
Not an issue, just wanted to thank you for this

### ISSUE THREAD

**Issue by tupinikeebs** [fabriciobastian/ploopy-adept-small-btu#1]
Heya, Not an issue at all I just wanted to thank you for this awesome job. I recently bought an Adept, and while I loved it, I found it too tall for my hand, somehow, I loved this version you made. Is there a way to "make room" to use the 44mm ball that comes with it while still being somewhat slim? also which BTU did you use?

**Comment by fabricio-bastian** [comment]
Hi there. Glad to hear that you liked it :)
I did consider using a larger ball while I was designing it. My choice of using a smaller ball was a bit about aesthetics, but also because, in my personal opinion, the 44mm ball has too much momentum on the BTUs. Also, it would compromise a bit on the usability of the buttons, because there is not that much room.
I think that the perfect size would be the ploopy nano trackball. Which is probably where I'm going next with this (I don't know when though :)). I just found it a bit hard to source trackballs on that size that look nice...

Regarding the BTUs, all the specs are in the README :)

**Comment by tupinikeebs** [comment]
I see, thank you again! I'll both print this one and try to change the ball size, I do have a few 34mm perixx here as well, But never used the btu's so I've no idea on how it affect the momentum.
I don't mind the trackball getting a bit bigger overall (I mean, in size, but not in height heh).

**Comment by fabricio-bastian** [comment]
Sounds great. Do you want access to the design file? I would love to see the results if you play with the design :)

I have used onshape and I can share it with you, so you can play with it. The CAD history is not particularly clean though 😅.

If you go for the 44mm ball, you will likely get something that is about as tall as the original Adept, with the exception of the front buttons, since this design is not slanted as the original is.

**Comment by tupinikeebs** [comment]
Yeah sure! I would love to have the design files! But now it makes sense, maybe the ball would be TOO high, and it wouldn't be that much usable, hmmmmmmmm, now I am not sure If it's a great idea! I was planning on using chinese BTU, which aren't that great, but are the ones I can buy hahahha. So I think maybe with a bigger ball, it won't have that much momentum.
 But I wasn't thinking about the height of the ball and how hard/easy it will be to push the buttons. damn, I'm dumb.

**Comment by fabriciobastian** [comment]
Hey again. I did a new version with a 1.5inch ball (same size as the nano), which you may like. I have uploaded the step files, which are easier to modify. The trackball is still quite low IMO, I can still use it in a natural hand position without any need for wrist rests. Check it out :)
